packages/lib-react-components/src/ZooFooter/ZooFooter.spec.js

```diff
@@ -5,7 +5,8 @@ import ZooFooter from './ZooFooter'
 
 
 describe('<ZooFooter />', function () {
-  let linkLists, socialLinks, policyNavigation
+  this.timeout(5000)
```

Comment:
I assume this is a workaround for the tests timing out randomly.

Reply:
User event tests take a while to run, so this to allow plenty of time for those. And the random timeouts too.